components/cd-flow/cd-flow.css

```diff
@@ -0,0 +1,25 @@
+/* https://piccalil.li/cube-css */
+
+/* CUBE CSS is a CSS methodology */
+/* orientated towards simplicity, pragmatism and consistency */
+/* Composition Utility Block Exception */
+
+.cd-flow > * + * {
+  margin-top: var(--cd-flow-space, 1rem);
+}
+
+.cd-flow > .visually-hidden + * {
+  margin-top: 0;
+}
```

Comment:
nice 💯

Reply:
actually now that I think about it, if `.visually-hidden` is not the first element, it introduce weird spacing. It might be worth doing:

```css
.cd-flow > .visually-hidden:first-child + * {
  margin-top: 0;
}
```